src/main/java/frc/robot/subsystems/Climber_Subsystem.java

```diff
@@ -87,73 +165,49 @@ public double getPitch() {
         return ahrs.getPitch();
     }
 
-    public boolean getVerticalRLimit() {
-        return verticalR.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
-    public boolean getVerticalLLimit() {
-        return verticalL.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
-    public boolean getHorizontalRLimit() {
-        return horizontalR.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
-    public boolean getHorizontalLLimit() {
-        return horizontalL.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
     public int getRVerticalPos() {
-        return verticalR.getSensorCollection().getQuadraturePosition() - oldRVertical;
+        int verticalPosition = verticalR.getSensorCollection().getQuadraturePosition();
+        int correctedPosition = verticalPosition - oldRVertical;
+        SmartDashboard.putNumber("Raw Right Vertical Position", verticalPosition);
+        SmartDashboard.putNumber("Right Vertical Position", correctedPosition);
+        return correctedPosition;
     }
 
     public int getLVerticalPos() {
-        return verticalL.getSensorCollection().getQuadraturePosition() - oldLVertical;
+        int verticalPosition = verticalL.getSensorCollection().getQuadraturePosition();
+        int correctedPosition = verticalPosition - oldLVertical;
+        SmartDashboard.putNumber("Raw Left Vertical Position", verticalPosition);
+        SmartDashboard.putNumber("Left Vertical Position", correctedPosition);
+        return correctedPosition;    
     }
 
     public int getRHorizontalPos() {
-        return horizontalR.getSensorCollection().getQuadraturePosition() - oldRHorizontal;
+        int horizontalPosition = horizontalR.getSensorCollection().getQuadraturePosition();
+        int correctedPosition = horizontalPosition - oldRHorizontal;
+        SmartDashboard.putNumber("Raw Right Horizontal Position", horizontalPosition);
+        SmartDashboard.putNumber("Right Horizontal Position", correctedPosition);
+        return correctedPosition;
     }
 
     public int getLHorizontalPos() {
-        return (-horizontalL.getSensorCollection().getQuadraturePosition()) - oldLHorizontal;
```

Comment:
This one actually did change. Mysteriously this one has a negative in it. I suspect this has something to do with things being out of phase. That would mean we couldn't leave it like this and get the soft limits working. To get this working properly we'll have to make sure that everything is counting the same direction.